packages/autocannon-ui-web/README.md

```diff
@@ -9,6 +9,6 @@ backend API and React frontend.
 To install and run the web application:
 
 ```sh
-npm install -g autocannon-ui-web
-autocannon-ui-web
+npm install -g autocannon-ui
+autocannon-ui
```

Comment:
it feels we're getting into a little bit of duplication here. I wonder, do we really need the nested autocannon-ui(-web) package in the first place? could the code that combines frontend and backend live in the root package so we just publish that one and we mark the others as private, thereby:

1. removing a nested package
2. removing from npm the supporting packages which we have only for the sake of simplifying our work on the code base?